### ISSUE TITLE
PyCharm Jupyter notebook support insists on inbuilt markdown plugin

### ISSUE THREAD

**Issue by obestwalter** [vsch/idea-multimarkdown#742]
Since the last release of PyCharm it insists on activating the inbuilt markdown plugin to be able to use the Jupyter notebook support. Is this problem known already?

**Comment by vsch** [comment]
@obestwalter, yes it is. There is an issue raised by another user, with my offer to make Markdown Navigator compatible: https://youtrack.jetbrains.com/issue/PY-34367

**Comment by obestwalter** [comment]
Thanks - added my two cents from a user perspective in the issue. I wish you plugin would be the standard markdown plugin in all Intellij tools. Keep up the great work!

**Comment by vsch** [comment]
@obestwalter, thank you. I am sure that a solution is possible if JetBrains is willing to put some effort on it.

I cannot see what Markdown Support offers that Markdown Navigator cannot have, especially if I am willing to provide code to make the adaptation as easy as possible.
